### ISSUE TITLE
Package Control of Sublime Text 3 is not working

### ISSUE THREAD

**Issue by LuisGeorge** [wbond/package_control#1342]
Please help, since weeks ago i tried to install the Package Control for later can install some plugins and but i can't do it. After to try of install it, this show in the console

Im using Windows 8

**Comment by LuisGeorge** [comment]
Please help, since one month ago i tried to install the Package Control to later install some plugins and but i can't do it.  After to try of install it, this show in the console
[package control not working.txt](https://github.com/wbond/package_control/files/1779011/package.control.not.working.txt)

**Comment by LuisGeorge** [comment]
DPI scale: 1
startup, version: 3143 windows x32 channel: stable
executable: /C/Program Files/Sublime Text 3/sublime_text.exe
working dir: /C/Program Files/Sublime Text 3
packages path: /C/Users/Maritza/AppData/Roaming/Sublime Text 3/Packages
state path: /C/Users/Maritza/AppData/Roaming/Sublime Text 3/Local
zip path: /C/Program Files/Sublime Text 3/Packages
zip path: /C/Users/Maritza/AppData/Roaming/Sublime Text 3/Installed Packages
ignored_packages: ["Vintage"]
pre session restore time: 1.50674
startup time: 1.62174
first paint time: 1.62174
reloading plugin Default.auto_indent_tag
reloading plugin Default.block
reloading plugin Default.comment
reloading plugin Default.convert_syntax
reloading plugin Default.copy_path
reloading plugin Default.delete_word
reloading plugin Default.detect_indentation
reloading plugin Default.duplicate_line
reloading plugin Default.echo
reloading plugin Default.exec
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading plugin Default.history_list
reloading plugin Default.indentation
reloading plugin Default.install_package_control
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_context_url
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.profile
reloading plugin Default.quick_panel
reloading plugin Default.run_syntax_tests
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.settings
reloading plugin Default.show_scope_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.swap_line
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.trim_trailing_white_space
reloading plugin Default.ui
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin 0_package_control_loader.00-package_control
reloading plugin Package Control.1_reloader
reloading plugin Package Control.2_bootstrap
reloading plugin Package Control.Package Control
plugins loaded
Package Control: No updated packages
>>> import urllib.request,os,hashlib; h = '6f4c264a24d933ce70df5dedcf1dcaee' + 'ebe013ee18cced0ef93d5f746d80ef60'; pf = 'Package Control.sublime-package'; ipp = sublime.installed_packages_path(); urllib.request.install_opener( urllib.request.build_opener( urllib.request.ProxyHandler()) ); by = urllib.request.urlopen( 'http://packagecontrol.io/' + pf.replace(' ', '%20')).read(); dh = hashlib.sha256(by).hexdigest(); print('Error validating download (got %s instead of %s), please try manual install' % (dh, h)) if dh != h else open(os.path.join( ipp, pf), 'wb' ).write(by)
286331
>>> 
>>> import urllib.request,os,hashlib; h = '6f4c264a24d933ce70df5dedcf1dcaee' + 'ebe013ee18cced0ef93d5f746d80ef60'; pf = 'Package Control.sublime-package'; ipp = sublime.installed_packages_path(); urllib.request.install_opener( urllib.request.build_opener( urllib.request.ProxyHandler()) ); by = urllib.request.urlopen( 'http://packagecontrol.io/' + pf.replace(' ', '%20')).read(); dh = hashlib.sha256(by).hexdigest(); print('Error validating download (got %s instead of %s), please try manual install' % (dh, h)) if dh != h else open(os.path.join( ipp, pf), 'wb' ).write(by)
286331
>>>

**Comment by FichteFoll** [comment]
There is no error on the console and the package seemingly loaded fine. What makes you think that the package isn't working? Have you tried running the "Install Package" command in the Command Palette? If that doesn't work, try again after enabling the debug setting to produce a debug log that you can paste here: https://packagecontrol.io/docs/troubleshooting#Enabling_the_Debug_Log

**Comment by LuisGeorge** [comment]
Thanks bro!, how it was so easy, It ios not a problem if i install all packages from the "Install Packages" command?

**Comment by wbond** [comment]
You will almost certainly run into performance issues if you try to install all packages available, since there are thousands of them.

Closing since this seems to be a non-issue.

**Comment by jdpalmquist** [comment]
For others on Arch Linux x64 who might have this problem, during installation of the OS itself, I forgot to make a "/etc/locale.conf" file.  Once I made this file then added the lines "LANG=en_US.UTF-8" and "LC_CTYPE=en_US.UTF-8" to that file and rebooted the system the Sublime Text 3 Package Manager worked after that. I had previously verified that locale.gen was properly set and tried re-running locale-gen command, but that didn't work. I had also tried adding the env-vars to my bash profile and vconsole files and that didn't work either.
